en/services/multi_gcs.md

```diff
@@ -0,0 +1,128 @@
+# Multi GCS protocol
+## Introduction
```

Comment:
Might be worth first starting with the "use case" _like_:

> This protocol allows a GCS to set itself as the controller of a MAVLink system (vehicle), making it easier and safer to share control over vehicles in mult-GCS environments.
Note that the API only provides semantics for determining an owner, and handing over ownership: it does not provide any security layer.